packages/uni_app/lib/controller/local_storage/database/app_restaurant_database.dart

```diff
@@ -46,6 +46,7 @@ class RestaurantDatabase extends AppDatabase<List<Restaurant>> {
           return Restaurant(
             restaurantId,
             map['name'] as String,
+            map['period'] as String,
```

Comment:
Because you modify the database table, increase its version. Since it's stills in v1, you need to create migration code. Look at #1380

Reply:
@HenriqueSFernandes this doesn't appear to have been properly fixed, or am I wrong? I don't see a version number bump anywhere.